npm/webpack-dev-server/package.json

```diff
@@ -33,6 +33,7 @@
     "@types/proxyquire": "^1.3.28",
     "@types/speed-measure-webpack-plugin": "^1.3.4",
     "@types/webpack-dev-server-3": "npm:@types/webpack-dev-server@^3",
+    "@types/webpack-dev-server-4": "npm:@types/webpack-dev-server@^4",
```

Comment:
If case you didn't realize. It looks like https://www.npmjs.com/package/@types/webpack-dev-server has been deprecated.

Reply:
oh nice It looks like it ships with 4 by default. I don't think I am using them anyway so we can remove them and maybe revert the changes to the server

Reply:
looks like that worked. removed in 880cfa1256